app/javascript/components/rooms/MeetingBadges.jsx

```diff
@@ -3,17 +3,31 @@ import { Stack } from 'react-bootstrap';
 import PropTypes from 'prop-types';
 import MeetingStatusBadge from './MeetingStatusBadge';
 import MeetingParticipantsBadge from './MeetingParticipantsBadge';
+import MeetingSharedBadge from './MeetingSharedBadge';
 
-export default function MeetingBadges({ count }) {
+export default function MeetingBadges({ count, shared, online }) {
   return (
     <Stack direction="horizontal" gap={0} className="room-card-badges">
-      <MeetingStatusBadge />
-      { count >= 1
-        && <MeetingParticipantsBadge count={count} />}
+      { online
+        && (
+          <>
+            <MeetingStatusBadge />
+            <MeetingParticipantsBadge count={count} />
```

Comment:
Are we sure that we'd always have the attendees count when the room is online for all BBB servers??
We can be pessimistic about this since it will not cause any added complexity or reduce readability where we show each badge if its related condition was met and any input is valid,

Reply:
Yeah agreed. It was like that before but I felt it was getting a bit crazy in there. I'll see what I can do.